exonum-java-binding/fakes/src/test/java/com/exonum/binding/fakes/services/service/TestServiceIntegrationTest.java

```diff
@@ -37,7 +37,7 @@
 
 class TestServiceIntegrationTest {
 
-  @Disabled // FIXME: disabled until ECR-3320.
+  @Disabled // FIXME: disabled until ECR-3359.
```

Comment:
Why 3359? Does this code create several indexes?

Reply:
Yes, it creates the "_test_map" _ProofMapIndex_ two times from the same _Fork_.